src/main/kotlin/no/nav/k9brukerdialogapi/ytelse/fellesdomene/Utbetalingsperiode.kt

```diff
@@ -52,6 +63,7 @@ class Utbetalingsperiode(
     ) = FraværPeriode(
```

Comment:
Fint du benytter tom konstruktør med hjelpemetoder.
Den vil da delvisFravær automatisk uten at du trenger å opprette.

Se her for eksempel: https://github.com/navikt/k9-punsj/pull/684/files#diff-f4150051b4f4a094255e47f3f7e925cab673aca5cfe9da654b81731a51054633R130

Reply:
Ja kan være greit å bruke det nå som det er lagt til 👍

Reply:
Bedre? :) Skjønner ikke helt var arbeidstiden kommer ifra, ser ut som att k9format setter planlagt arbeidstid til 7.5 så mappingen blir feil?

Reply:
Trooor du ikke trenger å bruke den delvisFravær() fordi medFravær(timerBorte) og .medNormalarbeidstid(normalArbeidstid) setter den automatisk.